Libplanet/Net/BoundPeer.cs

```diff
@@ -53,6 +54,53 @@ private BoundPeer(SerializationInfo info, StreamingContext context)
         public static bool operator !=(BoundPeer left, BoundPeer right) =>
             Operator.Weave(left, right);
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="BoundPeer"/> class from
+        /// comma-separated string.</summary>
+        /// <param name="peerInfo">A comma-seperated string have format {pubkey},{host},{port}.
+        /// </param>
+        /// <returns>A <see cref="BoundPeer"/> from given data.</returns>
+        public static BoundPeer ParsePeer(string peerInfo)
+        {
+            if (peerInfo is null)
+            {
+                throw new ArgumentNullException(nameof(peerInfo));
+            }
+
+            string[] tokens = peerInfo.Split(',');
+            if (tokens.Length != 3)
+            {
+                throw new ArgumentException(
+                    $"'{peerInfo}', should have format <pubkey>,<host>,<port>");
+            }
+
+            if (!(tokens[0].Length == 130 || tokens[0].Length == 66))
+            {
+                throw new ArgumentException(
+                    $"'{peerInfo}', a length of public key must be 130 or 66 in hexadecimal," +
+                    $" but the length of given public key '{tokens[0]}' doesn't.");
+            }
+
+            try
+            {
+                var pubKey = new PublicKey(ByteUtil.ParseHex(tokens[0]));
+                var host = tokens[1];
+                var port = int.Parse(tokens[2], CultureInfo.InvariantCulture);
+
+                // FIXME: It might be better to make Peer.AppProtocolVersion property nullable...
+                return new BoundPeer(
+                    pubKey,
+                    new DnsEndPoint(host, port));
+            }
+            catch (Exception e)
+            {
+                throw new ArgumentException(
+                    $"{nameof(peerInfo)} seems invalid. [{peerInfo}]",
+                    innerException: e
+                );
+            }
+        }
```

Comment:
Could you write unittest for this static method?

Reply:
Applied. https://github.com/planetarium/libplanet/pull/1240/commits/447decfb9057b44f4f1485fe0044ce0f1428ef5b